<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>move</class>
 <widget class="QMainWindow" name="move">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MoVe: Monitoreo Vehicular Automatico</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="time_var">
    <property name="geometry">
     <rect>
      <x>844</x>
      <y>4</y>
      <width>145</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_var_2">
    <property name="geometry">
     <rect>
      <x>844</x>
      <y>24</y>
      <width>145</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="VideoPlayer">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>44</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#VideoPlayer
{
background-color: gray;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>40</y>
      <width>317</width>
      <height>149</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Cargar archivo</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>32</y>
       <width>65</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Fuente</string>
     </property>
    </widget>
    <widget class="QComboBox" name="selectSource">
     <property name="geometry">
      <rect>
       <x>84</x>
       <y>28</y>
       <width>185</width>
       <height>29</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Streaming</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Archivo de video</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>80</y>
       <width>57</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Puerto</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="selectPort">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="accelerated">
      <bool>false</bool>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>116</y>
       <width>65</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Archivo</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="filePath">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>116</y>
       <width>193</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFilePath">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>272</x>
       <y>115</y>
       <width>25</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnPlaynPause">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>244</y>
      <width>101</width>
      <height>57</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>332</y>
      <width>101</width>
      <height>57</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReports">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>448</y>
      <width>41</width>
      <height>49</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>104</x>
      <y>412</y>
      <width>85</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reportes</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="parameterMenu">
    <property name="geometry">
     <rect>
      <x>348</x>
      <y>528</y>
      <width>297</width>
      <height>69</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Parametros</string>
    </property>
    <widget class="QRadioButton" name="rbtn_param_SN">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>28</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>S-N</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbtn_param_NS">
     <property name="geometry">
      <rect>
       <x>84</x>
       <y>28</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>N-S</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbtn_param_EO">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>28</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>E-O</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbtn_param_OE">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>28</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>O-E</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="txtMensajes">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>571</y>
      <width>329</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="textCarCount">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>12</y>
      <width>145</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>VEHICULOS:  0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>994</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionSalir</sender>
   <signal>triggered()</signal>
   <receiver>move</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
